index.js

```diff
@@ -633,6 +662,17 @@ input UpdateMaxValueInput {
   custom: Int
 }
 
+input UpdateEarliestDateInput {
+  offset: DurationInput
+  relativePosition: RelativePosition
```

Comment:
Feels like `offset` and `relativePosition` would always have some value. Should they be mandatory in the schema to reflect this?